### PR TITLE
decorating chost-related tests due to BZ#1238247

### PR DESCRIPTION
**Pull request by rplevka**
addresses https://github.com/SatelliteQE/robottelo/issues/3964

### CODE REVIEW

File: tests/foreman/cli/test_import.py [hunk 1]
```diff
@@ -1589,7 +1590,7 @@ def test_negative_reimport_chosts(self):
                 )
                 clean_transdata()
 
-    @skip_if_bug_open('bugzilla', 1267224)
```
Comment (lpramuk) [inline tests/foreman/cli/test_import.py]: dont replace this one decoration, it's another bug in NEW state
instead add 2nd for bz 1238247

**Comment by rplevka** [comment]
comment addressed. Thanks for noticing

**Comment by lpramuk** [comment]
glob ```/.transition_data/system*``` stopped to work in 6.2 as ```systems``` got renamed to ```hosts```.

Thus:
system_content_views (OK)
system_activation_keys (OK)
systems >>> hosts (missed to handle)

New glob ```/.transition_data/{system*,hosts*}```

**Comment by lpramuk** [comment]
ACK

**Comment by sghai** [comment]
ACK

**Pull request merged**